### PR TITLE
Issue 349 buildpack and clusterbuildpack ref

### PR DESCRIPTION
**Pull request by mailprak**
Issue : #349

### CODE REVIEW

**Comment by chenbh** [comment]
@mailprak #349 is referring to the fact that the [`.spec.order.group`](https://github.com/buildpacks-community/kpack/blob/main/docs/builders.md#order) of a Builder can refer to a [Cluster]Buildpack object reference (eg `{"name": "my-buildpack","kind":"ClusterBuildpack"}`).

We need to display the [Cluster]Buildpacks that are _actually used_ by the Builder instead of every single one on the cluster.

**Comment by chenbh** [comment]
I forgot to mention it in the original issue, but do you mind also doing the same thing for `kp clusterbuilder status` as well? The only difference is that the ClusterBuilder will only have ClusterBuildpack refs

**Comment by mailprak** [comment]
> I forgot to mention it in the original issue, but do you mind also doing the same thing for `kp clusterbuilder status` as well? The only difference is that the ClusterBuilder will only have ClusterBuildpack refs

sure

File: pkg/commands/builder/status.go [hunk 1]
```diff
@@ -134,6 +134,23 @@ func printBuilderReadyStatus(bldr *v1alpha2.Builder, writer io.Writer) error {
 		return err
 	}
 
+	cpTableWriter, err := commands.NewTableWriter(writer, "BuildpackName", "     BuildpackKind")
```
Comment (chenbh) [inline pkg/commands/builder/status.go:137]: nit: the column headers should be more readable `Buildpack Name`, `Buildpack Kind`

File: pkg/commands/builder/status.go [hunk 2]
```diff
@@ -134,6 +134,23 @@ func printBuilderReadyStatus(bldr *v1alpha2.Builder, writer io.Writer) error {
...
+		for _, ref := range entry.Group {
+			if ref.ObjectReference.Name != "" && ref.ObjectReference.Kind != "" {
+			err := cpTableWriter.AddRow(ref.ObjectReference.Name, ref.ObjectReference.Kind)
+		if err != nil {
+			return err
+		}
+	}
+	 }
+	 cpTableWriter.Write()
+	}
```
Comment (chenbh) [inline pkg/commands/builder/status.go:152]: Can you run `gofmt` on your files? I think there's some weird whitespace padding in here

File: pkg/commands/clusterbuilder/status.go [hunk 3]
```diff
@@ -127,6 +127,23 @@ func printBuilderReadyStatus(bldr *v1alpha2.ClusterBuilder, writer io.Writer) er
 		return err
 	}
 
+	cpTableWriter, err := commands.NewTableWriter(writer, "ClusterBuildpackName", "     ClusterBuildpackKind")
```
Comment (chenbh) [inline pkg/commands/clusterbuilder/status.go:130]: nit: spaces in column names

**Review by chenbh — CHANGES_REQUESTED** [review]

**Comment by chenbh** [comment]
Sorry for the delay, added some comments

File: pkg/commands/builder/status.go (hunk 1 above)
Comment (mailprak) [reply pkg/commands/builder/status.go]: Done

File: pkg/commands/builder/status.go (hunk 2 above)
Comment (mailprak) [reply pkg/commands/builder/status.go]: Done

File: pkg/commands/clusterbuilder/status.go (hunk 3 above)
Comment (mailprak) [reply pkg/commands/clusterbuilder/status.go]: Done

**Review by chenbh — APPROVED** [review]
Thanks!

**Pull request merged**